common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5006,6 +5011,11 @@
           "Multiple bucket TRANSFORMs."
         ]
       },
+      "MULTIPLE_QUERY_RESULT_CLAUSES_WITH_PIPE_OPERATORS" : {
+        "message" : [
+          "Syntax error: the SQL pipe operator syntax using |> does not support <clauses>. Please separate the multiple result clauses into separate pipe operators and then retry the query again."
```

Comment:
how about
```
<clause1> and <clause2> can not coexist in SQL pipe operator (1>). Please separate ...
```

Reply:
The idea is to make the error parameters simple, so that people can consume it programatically.

Reply:
Sure, switched to this format.